Makefile

```diff
@@ -0,0 +1,11 @@
+all:
+	true
+
+install:
+	install -D firefox /app/bin/firefox
+	install -D stencyl.sh /app/bin/stencyl
+	install -D com.stencyl.Game.desktop /app/share/applications/com.stencyl.Game.desktop
+	install -D com.stencyl.Game.png /app/share/icons/hicolor/128x128/apps/com.stencyl.Game.png
+	install -D com.stencyl.Game.appdata.xml /app/share/appdata/com.stencyl.Game.appdata.xml
+	ln -s /usr/lib/libpcre.so.1 /app/lib/libpcre.so.3
```

Comment:
Maybe add a comment that stencil is a closed source software, so we couldn't build it. Instead we got a binary package which was likely built on Ubuntu, where the pcre library is installed as `libpcre.so.3`, whereas the similar, API (and hopefully ABI-) compatible pcre in the flatpak runtime is `libpcre.so.1`.